packages/web3torrent/.env

```diff
@@ -3,10 +3,13 @@
 SKIP_PREFLIGHT_CHECK=true
 
 REACT_APP_WALLET_URL=http://localhost:3055
-REACT_APP_FAKE_CHANNEL_PROVIDER=false
+REACT_APP_FAKE_CHANNEL_PROVIDER=true
 
 REACT_APP_CHAIN_NETWORK_ID=9001
 CHAIN_NETWORK_ID=9001
 
 REACT_APP_FIREBASE_PROJECT='rock-paper-scissors-dev-4ff8d'
-REACT_APP_FIREBASE_API_KEY='AIzaSyAOvhDzJir_El3O6SJ2xQlrpOisnObq6zw'
\ No newline at end of file
+REACT_APP_FIREBASE_API_KEY='AIzaSyAOvhDzJir_El3O6SJ2xQlrpOisnObq6zw'
+
+REACT_APP_TRACKER_URL=web3torrent-tracker-staging.herokuapp.com
```

Comment:
what's the significance of the `REACT_APP` prefix here?

Reply:
It's necessary so that the React app can access it.
See [here](https://create-react-app.dev/docs/adding-custom-environment-variables/)